bindings/python/test.py

```diff
@@ -230,7 +230,7 @@ def test_read_resume_data(self):
         tp = lt.read_resume_data(resume_data)
 
         self.assertEqual(tp.name, 'test')
-        self.assertEqual(tp.info_hash, lt.sha1_hash('abababababababababab'))
+        self.assertEqual(tp.info_hash.v1, lt.sha1_hash('abababababababababab'))
```

Comment:
I'm a little bit worried about this API break in python. Is there an easy way to do implicit conversions in python?